install_ant-media-server.sh

```diff
@@ -59,6 +59,25 @@ restore_settings() {
   fi
 }
 
+distro () {
+  os_release="/etc/os-release"
+  if [ -f "$os_release" ]; then
+    . $os_release
+    msg="We are supporting Ubuntu 16.04, 18.04 and Centos 7."
+    if [ "$ID" == "ubuntu" ] || [ "$ID" == "centos" ]; then  
```

Comment:
What about supporting Red Hat? I think when we support Centos, we have some support for Red Hat

Reply:
Yes, It 's the same. We need to test it.